ls1intum/Artemis
# [Refactor] improved FileUploadExerciseResource

### Checklist

### Description

adapted FileUploadExercise to new Rest Security Guidelines
### Steps for Testing


1. Log in to Artemis as Editor or higher
2. create a FileUploadExercise
3. edit the exercise after creation
4. verify that the FileUploadExercises get displayed in the Courses ExerciseOverView
5. look at the ExerciseDetailPage of a FileUploadExercise
6. re-evaluate a FileUplaodExercise (probably need some submissions for that)
7. delete your created exercise (need to be Instructor for this)

### Review Progress





- Code Review
- Manual Tests


### Test Coverage




FileUploadExerciseResource: 99%
